README.md

```diff
@@ -1,2 +1,2 @@
-# Survey-Form-Trial
-This is a survey form that has been done in the trial.
+# Landing-Page-Trial
+This is a landing page that has been done in the trial.
```

Comment:
Please add descriptive README file to your project - please use this template but remember to customize it to your project. It will make your repo professional.